invokeai/backend/install/check_root.py

```diff
@@ -32,5 +33,6 @@ def check_invokeai_root(config: InvokeAIAppConfig):
         print(
             '** From the command line, activate the virtual environment and run "invokeai-configure --yes --skip-sd-weights" **'
         )
+        print('** (To skip this check completely, add "--ignore_missing_core_models" to your CLI args. Not recommended.)')
```

Comment:
Could you change the user message to tell them the consequences of skipping this check? 

"Not installing these core models will prevent the loading of some or all .safetensors and .ckpt files. You can always come back and install these core models in the future."

Reply:
done